cookieDriver.js

```diff
@@ -0,0 +1,290 @@
+/*!
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import {get, getAll, remove, set} from 'tiny-cookie';
+
+export default {
+    _driver: 'cookieWrapper',
+    _initStorage,
+    iterate,
+    getItem,
+    setItem,
+    removeItem,
+    clear,
+    length,
+    key,
+    keys,
+    dropInstance
+};
+
+function _initStorage(options) {
+  const dbInfo = {};
+  if(options) {
+    Object.assign(dbInfo, options);
+  }
+  dbInfo.keyPrefix = _getKeyPrefix(options, this._defaultConfig);
+  // FIXME: support `serializer`?
+  //dbInfo.serializer = serializer;
+  this._dbInfo = dbInfo;
+  return Promise.resolve();
+}
+
+async function clear(callback) {
+  try {
+    await this.ready();
+    _clear(this._dbInfo.keyPrefix);
+  } catch(e) {
+    if(callback) {
+      return callback(e);
+    }
+    throw e;
+  }
+  if(callback) {
+    callback();
+  }
+}
+
+async function getItem(key, callback) {
+  let result;
+  key = _normalizeKey(key);
+
+  try {
+    await this.ready();
+    result = get(this._dbInfo.keyPrefix + key);
+    if(result) {
+      // FIXME: support `dbInfo.serializer`?
+      result = JSON.parse(result);
+    }
+  } catch(e) {
+    if(callback) {
+      return callback(e);
+    }
+    throw e;
+  }
+  if(callback) {
+    callback(null, result);
```

Comment:
add return for safety.  Addition instance of this pattern are not commented on.

Reply:
See above -- and it's the same for the others. I copied what the other drivers from the `localForage` lib do.